worldedit-core/src/main/java/com/sk89q/worldedit/util/formatting/component/MessageBox.java

```diff
@@ -33,7 +33,12 @@
  */
 public class MessageBox extends TextComponentProducer {
 
-    private static final int GUARANTEED_NO_WRAP_CHAT_PAGE_WIDTH = 47;
+    //decremented by one (from 47) to account for minecraft font characters having varying widths
+
+    // method for applying border needs to be replaced in the future  - need to
+    // be able to measure length of title (and prefix) by pixel width, not by character width
+    // (to avoid wrapping in case of thick chars, or not enough spacers in case of thin chars)
+    private static final int GUARANTEED_NO_WRAP_CHAT_PAGE_WIDTH = 46;
```

Comment:
I'd like to keep full compatibility here with WE. If this is an actual issue, it should be addressed in WE instead.

Reply:
alright, i'll see what I can do